gremlin-python/src/main/python/gremlin_python/structure/io/graphbinaryV1.py

```diff
@@ -255,15 +257,15 @@ class LongIO(_GraphBinaryTypeIO):
     @classmethod
     def dictify(cls, obj, writer, to_extend, as_value=False, nullable=True):
         if obj < -9223372036854775808 or obj > 9223372036854775807:
-            raise Exception("TODO: don't forget bigint")
+            raise Exception("Value too big, please use bigint Gremlin type")
```

Comment:
Should we just promote the type to bigint here? That way we handle this implictly and explicitly. Meaning you could do something like:

`g.V().....property("bigint_explicit", bigint(1)).property("bigint_implicit", 9223372036854775807 + 1)`.

Reply:
implicit conversion can introduce new issues, for example 
`g.inject(1).repeat(__.addV('person').property('name', __.loops())).times(9223372036854775807 + 1).count()`

leads to server exception
`[ERROR] TraversalOpProcessor - Could not deserialize the Traversal instance
java.lang.IllegalStateException: null: .times(BigInteger)`

if you think that this is not an issue, then I will add implicit conversion

Reply:
Fair point. I am not sure myself I will tag some people who might know - @krlawrence @spmallette any thoughts?